docker/docker-compose.yml

```diff
@@ -92,7 +92,7 @@ services:
           "--no-verbose",
           "--tries=1",
           "--spider",
-          "http://localhost:9999/health"
+          "http://auth:9999/health"
```

Comment:
**remark**

`wget` doesn't necessarily love the "localhost", while `curl` will translate it to `127.0.0.1` and pass, `wget` depending on versions will error out. So I replaced all the mentions of localhost by the container network name so the behavior remain consistent.